### ISSUE TITLE
Smithing slots and some recipes misbehave on forge on 2.6

### ISSUE THREAD

**Issue by sisby-folk** [sisby-folk/tinkerers-smithing#31]
Some items can't be placed in smithing tables, like stone and iron pickaxes (but wood and diamond pickaxes can?)

Some recipes don't seem to generate properly as well - seemingly additions that aren't tags? no logs.

```
Registered 7 Tool Materials with 36 items: [minecraft:diamond(5), minecraft:string(3), minecraft:gold(5), minecraft:stone(5), minecraft:wood(6), minecraft:iron(7), minecraft:netherite(5)]
Registered 7 Armor Materials with 25 items: [minecraft:diamond(4), minecraft:chain(4), minecraft:gold(4), minecraft:leather(4), minecraft:turtle(1), minecraft:iron(4), minecraft:netherite(4)].
Registered 14 Equipment Types with 60 items: [minecraft:hoe(6), minecraft:shield(1), minecraft:fishing_rod(1), minecraft:shovel(6), minecraft:helmet(7), minecraft:sword(6), minecraft:bow(1), minecraft:shears(1), minecraft:axe(6), minecraft:leggings(6), minecraft:pickaxe(6), minecraft:crossbow(1), minecraft:boots(6), minecraft:chestplate(6)]
Applied 38 Unit Costs to 37 items
Applied 40 Upgrade Paths to 27 items
Applied 0 Sacrifice Paths to 0 items
Added 56 runtime recipes with 0 data overrides!
```

vs fabric 1.20:

```
Registered 7 Tool Materials with 36 items: [minecraft:diamond(5), minecraft:string(3), minecraft:gold(5), minecraft:stone(5), minecraft:wood(6), minecraft:iron(7), minecraft:netherite(5)]
Registered 7 Armor Materials with 25 items: [minecraft:diamond(4), minecraft:chain(4), minecraft:gold(4), minecraft:leather(4), minecraft:turtle(1), minecraft:iron(4), minecraft:netherite(4)].
Registered 14 Equipment Types with 58 items: [minecraft:hoe(6), minecraft:shield(1), minecraft:fishing_rod(0), minecraft:shovel(6), minecraft:helmet(7), minecraft:sword(6), minecraft:bow(1), minecraft:shears(1), minecraft:axe(6), minecraft:leggings(6), minecraft:crossbow(0), minecraft:pickaxe(6), minecraft:boots(6), minecraft:chestplate(6)]
Applied 67 Unit Costs to 66 items
Applied 40 Upgrade Paths to 27 items
Applied 9 Sacrifice Paths to 9 items
Added 210 runtime recipes with 0 data overrides!
```

**Comment by sisby-folk** [comment]
unmarked 2.6 as forge compatible for now.

**Comment by sisby-folk** [comment]
Still the same on 2.6.6

**Comment by sisby-folk** [comment]
seems like only recipes with tag ingredients work.

**Comment by sisby-folk** [comment]
tldr 
![image](https://github.com/sisby-folk/tinkerers-smithing/assets/55819817/9a4e2206-ee5c-459b-8119-84cd9f40df8a)
